Change: valueC => valueX

@package vaimo/composer-patches-target1
@assert vaimo/composer-patches-target1,valueC,valueX

--- src/example.txt.org
+++ src/example.txt
@@ -5,7 +5,7 @@
 value5
 value6
 value7
-valueC
+valueX
 value9
 value10
 value11
